partcad/tests/unit/test_adhoc_convert.py

```
@@ -0,0 +1,120 @@
+    source_file = Path(f"examples/feature_convert/{input_format}/{TEST_FILES[input_format]}")
+    test_file = tmp_path / TEST_FILES[input_format]
+
+    if not source_file.exists():
+        pytest.skip(f"Skipping: Test file {source_file} not found.")
+
+    test_file.write_bytes(source_file.read_bytes())  # Copy the file for testing
+    return test_file, input_format
+
+
+@pytest.fixture(params=OUTPUT_FORMATS)
+def temp_output_file(tmp_path, request):
```

Ruff (0.8.2)

Missing return type annotation for public function `temp_output_file`

(ANN201)